packages/sui-bundler/README.md

```diff
@@ -153,19 +146,14 @@ Puedes configurar que quieres cachear para ser usado offline:
 }
 ```
 
-runtime usa la mismo API que (sw-toolbox)[https://googlechrome.github.io/sw-toolbox/]
-
-además whitelist es un listado de regexp que indican que páginas son seguras para hacer solo CSR. Puedes usar el string `::all::` para indicar que siempre quieres usar CSR.
-
-Si no defines un listado whitelist. Siempre se hará SSR
+Runtime follows the (API of sw-toolbox)[https://googlechrome.github.io/sw-toolbox/]. Also, the whitelist is a list of regexp that indicates which pages are secure to use only Client Server Rendering. You could use the `::all::` string for indicating that you always want to use Client Side Rendering.
```

Comment:
FYI dead link: use https://github.com/GoogleChromeLabs/sw-toolbox

"to use only Client" => "to use only with Client" ?

string for indicating that you always want to use Client Side Rendering. => string  to indicate that you always want to use...